pkgs/development/python-modules/expecttest/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+
+# Native build inputs
+, poetry-core
+
+# Check inputs
+, python
+, hypothesis
+}:
+
+buildPythonPackage rec {
+  pname = "expecttest";
+  version = "0.1.3";
```

`4be7413` doesn't point to 0.1.3.

```suggestion
  version = "unstable-2021-08-30";
```

---

Sorry about that, is there a way to check which version pypi points to? (This repo doesn't have any git tags)

---

Altered the rev to what looks to be the official release rev https://github.com/ezyang/expecttest/commit/1450e7fc26d5a55a2efb8cca93b0f193a259d8e1